src/geometry/frustum.rs

```diff
@@ -12,30 +12,22 @@ use serde::{Deserialize, Serialize};
 /// creating the perspective projection, see also the frustum unit test below.
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct Frustum<S: BaseFloat> {
-    query_from_eye: Isometry3<S>,
-    clip_from_eye: Perspective<S>,
-    query_from_clip: Matrix4<S>,
+    world_from_clip: Matrix4<S>,
```

Comment:
I'm not 100% sure, whether the renaming here makes it clearer, in particular in situations where the Frustum is not defined in world coordinates, but some local coordinates...

Reply:
Ok, I'll undo it

Reply:
Done.